code/controllers/subsystem/processing/fields.dm

```diff
@@ -0,0 +1,6 @@
+PROCESSING_SUBSYSTEM_DEF(fields)
```

Comment:
Why not just hook into `SSfastprocess`? They are both processing subsystems, and they have the same wait.

Reply:
I thought about that and wasn't sure which to use. TG uses `SSfields`, so thats what I used. I assume its for the `SS_KEEP_TIMING` and some of the other flags.